backend/src/jest/helpers.js

```diff
@@ -15,3 +15,9 @@ export async function login(variables) {
     authorization: `Bearer ${response.login}`,
   }
 }
+
+//* This is a fake ES2015 template string, just to benefit of syntax
+// highlighting of `gql` template strings in certain editors.
+export function gql(strings) {
+  return strings.join('')
+}
```

Comment:
I like this solution, but how do you feel about it @roschaefer 
don't want you to feel like you are making too many concessions
I am also ok dealing with no syntax highlighting, not sure how @Tirokk feels.